src/main/scala/spinoco/fs2/http/HttpClient.scala

```diff
@@ -114,12 +113,10 @@ trait HttpClient[F[_]] {
   def apply[F[_] : ConcurrentEffect : ContextShift : Timer](
    requestCodec         : Codec[HttpRequestHeader]
```

Comment:
Do we need still the timer?

Reply:
Yes, it is required for the websocket client, as we need to do pings there.